src/Kuzzle.js

```diff
@@ -523,11 +526,11 @@ Kuzzle.prototype.login = function (strategy) {
 
   if (typeof credentials === 'object') {
     Object.keys(credentials).forEach(function (key) {
```

Comment:
The body could just be set to credentials instead of copying it property by property